packages/next/next-server/lib/constants.ts

```diff
@@ -9,6 +9,7 @@ export const EXPORT_DETAIL = 'export-detail.json'
 export const PRERENDER_MANIFEST = 'prerender-manifest.json'
 export const ROUTES_MANIFEST = 'routes-manifest.json'
 export const REACT_LOADABLE_MANIFEST = 'react-loadable-manifest.json'
+export const FONT_MANIFEST = 'font-manifest.json'
```

Comment:
Instead of adding a new manifest it'd easier to reuse the build-manifest I think 🤔

Reply:
May be, however I am afraid that by writing `build-manifest` from multiple plugins they might overwrite each other?

Reply:
It doesn't have to be multiple plugins though, we can share information into the build manifest 👍

The reason using build manifest is better is that it's automatically made available to rendering and _document, meaning you don't have to change the rendering code as much as you have to when introducing a new file

Reply:
I found that BuildManifestPlugin is built on client run of webpack. This is a problem because FontStylesheetGatheringPlugin runs on the server build of webpack.
The reason for keeping it on server side run is that otherwise it does not build the user defined `_document.jsx` which is a common place to define stylesheets and fonts and seems to be building only for server run of webpack.

Reply:
@timneutkens would there be a way to generate this on server run and consolidate during client bundle of webpack?